core/test/unit/utils/image-size_spec.js

```diff
@@ -0,0 +1,89 @@
+var should = require('should'),
+    Promise = require('bluebird'),
+    nock = require('nock'),
+    result,
+
+    // Stuff we are testing
+    imageSize = require('../../../server/utils/image-size');
+
+describe('Image Size', function () {
+    it('should have an image size function', function () {
+        should.exist(imageSize);
+    });
+
+    it('should return image dimensions with http request', function (done) {
```

Comment:
@kirrg001 : I'm not sure, if I should rather nock all the requests? What do you think? My internet connection now for example is really bad and in that case, this assert and the next one fail, because the timeout kicks in.